docs/source/get_started/summary.md

```diff
@@ -0,0 +1,33 @@
+# TL;DR
+
+This page summarises what you've learned about Kedro so far.
+
+## Logistics
+* Kedro can be used on Windows, macOS or Linux
+* [Installation prerequisites](install.md#installation-prerequisites) include Python 3.7+, `git` and `conda`
+* You should install Kedro using `pip install kedro`
+
+## Kedro fundamentals
+
+* Kedro nodes are the building blocks of data processing pipelines. A node is a wrapper for a Python function that names the inputs and outputs of that function.
+* A pipeline organises the dependencies and execution order of a collection of nodes.
+* Kedro has a registry of all data sources the project can use called the Data Catalog. There is inbuilt support for various file types and file systems.
+* Kedro projects follow a default template that uses specific folders to store datasets, notebooks, configuration and source code.
+
+
+## Kedro project creation
+* You can create a Kedro project:
+    * with just the basic code: `kedro new`
+    * or you can populate a new project with pre-built code, e.g. `kedro new --starter=pandas-iris` from a [range of starter projects](../kedro_project_setup/starters.md#list-of-official-starters)
+* Once you've created a project, you need to navigate to its project folder; you can then install its dependencies: `pip install -r src/requirements.txt`
+* To run the project: `kedro run`
+* To visualise the project: `kedro viz`
```

Comment:
You're right, and @noklam raised this too. I am updating the `pandas-iris` starter to add kedro-viz to dependencies. It seems a bit odd to create a dependency to serve the docs, but I think it's an acceptable addition because many/most project users will want it. Make sense @merelcht ?

Reply:
Yes it does, but if the user opts to create an empty project and not follow the starter instructions, then they still wouldn't have viz. I guess it's likely that most people would follow the starter instructions and not go into the empty project straight away, so it's very minor.